src/ucp/rndv/proto_rndv.c

```diff
@@ -475,9 +475,15 @@ void ucp_proto_rndv_ctrl_probe(const ucp_proto_rndv_ctrl_init_params_t *params,
             /* Consider remote priority and threshold only if RNDV_SCHEME or
              * BCOPY/ZCOPY thresh are set to force these settings */
             cfg_priority = remote_proto->cfg_priority;
-            cfg_thresh   = ucs_max(cfg_thresh, remote_proto->cfg_thresh);
+            cfg_thresh   = (cfg_thresh == UCS_MEMUNITS_AUTO) ?
```

Comment:
can it be `INF`?

Reply:
It can be inf only in case when RNDV_THRESH=inf, so it is OK to take this value